stream/deduper.js

```diff
@@ -0,0 +1,18 @@
+var peliasConfig = require( 'pelias-config' ).generate();
+var addressDedupeStream = require( 'pelias-address-deduplicator' );
+
+
+function createStream() {
+  // if deduping is enabled, create the deduper stream
+  // otherwise create a passthrough stream that does nothing
+  if (peliasConfig.imports.openstreetmap.deduplicate) {
+    return addressDedupeStream();
+  }
+  else {
```

Comment:
This is cool. The pass-through streams makes the main code that assembles the pipes much cleaner.